README.rst

```diff
@@ -295,24 +307,40 @@ The C++ code is tested with
 
 .. installation_LKokkos-start-inclusion-marker-do-not-remove
 
+-----------------------------
+
 Lightning-Kokkos installation
 *****************************
 
 On linux systems, ``lightning.kokkos`` with the OpenMP backend can be installed by providing the optional ``[kokkos]`` tag:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning[kokkos]
+    pip install pennylane-lightning[kokkos]
 
 Install Lightning-Kokkos from source
 ====================================
 
 As Kokkos enables support for many different HPC-targeted hardware platforms, ``lightning.kokkos`` can be built to support any of these platforms when building from source.
 
+Install Kokkos (Optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
 We suggest first installing Kokkos with the wanted configuration following the instructions found in the `Kokkos documentation <https://kokkos.github.io/kokkos-core-wiki/building.html>`_.
 For example, the following will build Kokkos for NVIDIA A100 cards
 
-.. code-block:: console
+Download the `Kokkos code <https://github.com/kokkos/kokkos/releases>`_. Lightning Kokkos was tested with Kokkos version <= 4.3.01  
+
+.. code-block:: bash
+
+    # Replace x, y, and z by the correct version
+    wget https://github.com/kokkos/kokkos/archive/refs/tags/4.x.yz.tar.gz 
+    tar -xvf 4.x.yz.tar.gz
+    cd kokkos-4.x.yz
+
+Build Kokkos for NVIDIA A100 cards
```

Comment:
What about Nvidia GPU with `SM80` capability?

Reply:
I think it is easier to recognize the name NVIDIA A100 than `SM80` but it could be something like this 
``` text 
Build Kokkos for NVIDIA A100 cards (SM80 architecture)
```
What do you think?